regulations/static/regulations/js/source/helpers.js

```diff
@@ -38,6 +38,17 @@ if (!Array.prototype.indexOf) {
   };
 }
 
+$(window).load(function () {
```

Comment:
`onload`s like this don't work when content is ajaxed in -- doesn't affect us here because the only page that currently uses it isn't ajaxed in. `onload`s are also frowned on because they're difficult to track; think of them like global state. Is there another view (maybe the `MainView`?) that this belongs in, instead?